troi/tools/apple_lookup.py

```diff
@@ -25,7 +14,16 @@ def get_tracks_from_apple_playlist(developer_token, user_token, playlist_id):
         response = response.json()
```

Comment:
Does the apple music API have any sort of rate-limiting we might need to worry about here?